### DISCUSSION TITLE
Publications section

### DISCUSSION THREAD

**Discussion by christos-vasileiou** [arifszn/gitprofile#515] [Feature requests]
Is there a possiblity to add a section for papers and journals publications?

**Comment by arifszn** [comment]
Can you share an example of what the section should have? For example: title, link, description?

**Comment by christos-vasileiou** [comment]
Yes. More or less, something similar to certification section. Bullets (or numbers) where you can write: 
1. the conference that the paper or journal was published, 
2. authors' names
3. the title 
4. A link to the paper, 
5. A link to the code implementation (optional). 
6. The description (optional).

**Comment by arifszn** [comment]
Tracking in #519.
